test/fail_compilation/failcstuff1.d

```diff
@@ -0,0 +1,8 @@
+// EXTRA_FILES: imports/cstuff1.c
+/* TEST_OUTPUT
+---
+fail_compilation/imports/cstuff1.c(5): Error: no members for `enum empty_enum`
+fail_compilation/imports/cstuff1.c(6): Error: no members for anonymous enum
+---
```

Comment:
Windows boxen fail with:
```
expected:
----
fail_compilation/imports/cstuff1.c(5): Error: no members for `enum empty_enum`
fail_compilation/imports/cstuff1.c(6): Error: no members for anonymous enum
----
actual:
----
fail_compilation\imports\cstuff1.c(5): Error: no members for `enum empty_enum`
fail_compilation\imports\cstuff1.c(6): Error: no members for anonymous enum
```

urgh.

Reply:
The test script probably does not apply the path-rewrite-regex on .c files.